### PR TITLE
[zh] Sync contribution guidelines (5)

### PR DESCRIPTION
**Pull request by tengqm**
This PR is about reviews.

### CODE REVIEW

File: content/zh/docs/contribute/review/for-approvers.md [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+- The PR author explicitly disallows edits from approvers.
+-->
+如果处于下列情况之一，你不可以向别人的 PR 提交内容：
+
+- 如果 PR 作者是直接将自己的分支提交到
+  [https://github.com/kubernetes/website/](https://github.com/kubernetes/website/)
+  仓库。只有具有推送权限的评阅人才可以向他人的 PR 提交内容。
+
+  {{< note >}}
+  我们应鼓励作者下次将分支推送到自己的克隆副本之后在发起 PR。
```
Comment (SataQiu) [inline content/zh/docs/contribute/review/for-approvers.md:125]:
```suggestion
  我们应鼓励作者下次将分支推送到自己的克隆副本之后再发起 PR。
```

File: content/zh/docs/contribute/review/for-approvers.md [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+### 评判 Issue {#triaging-an-issue}
+
+1. 验证 Issue 的合法性
+
+  - 确保 Issue 是关于网站文档的。某些 Issue 可以通过回答问题或者为报告者提供
+    资源链接来快速关闭。
+    参考[请求支持或代码缺陷报告](#support-requests-or-code-bug-reports)
+    节以了解详细信息。
+  - 评估该 Issue 是否有价值。
+  - 如果 Issue 缺少足够的细节以至于无法采取行动，或者报告者没有通过模版提供给
```
Comment (SataQiu) [inline content/zh/docs/contribute/review/for-approvers.md:218]:
```suggestion
  - 如果 Issue 缺少足够的细节以至于无法采取行动，或者报告者没有通过模版提供
```

File: content/zh/docs/contribute/review/for-approvers.md [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+  `priority/awaiting-more-evidence` | 占位符，标示 Issue 可能是一个不错的 Issue，避免该 Issue 被忽略或遗忘
+  `help` or `good first issue` | 适合对 Kubernetes 或 SIG Docs 经验较少的贡献者来处理。更多信息可参考[需要帮助和入门候选 Issue 标签](https://github.com/kubernetes/community/blob/master/contributors/guide/help-wanted.md)。
+  {{< /table >}}
+
+   基于你自己的判断，你可以选择某 Issue 来处理，为之发起 PR
+   （尤其是那些可以很快处理或与你已经在做的工作相关的 Issue）。
+
+如果你对 Issue 评判有任何问题，可以在 `#sig-docs` Slack 频道或者
+[kubernetes-sig-docs 邮件列表](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
+中提问。
```
Comment (SataQiu) [inline content/zh/docs/contribute/review/for-approvers.md:262]: 预览时，该节没有正常显示出来，https://deploy-preview-22246--kubernetes-io-master-staging.netlify.app/zh/docs/contribute/review/for-approvers/#triaging-an-issue

File: content/zh/docs/contribute/review/for-approvers.md [hunk 4]
```diff
@@ -0,0 +1,432 @@
...
+
+SIG Docs 常常会遇到以下类型的 Issue，因此对其处理方式描述如下。
+
+### 重复的 Issue {#duplicate-issues}
+
+如果针对同一个问题有不止一个打开的 Issue，可以将其合并为一个 Issue。
+你需要决定保留哪个 Issue 为打开状态（或者重新登记一个新的 Issue），
+然后将所有相关的信息复制过去并提供对关联 Issues 的链接。
+最后，将所有其他描述同一问题的 Issue 标记为 `triage/duplicate` 并关闭之。
+保持只有一个 Issue 待处理有助于减少困惑，避免在统一问题上发生重复劳动。
```
Comment (SataQiu) [inline content/zh/docs/contribute/review/for-approvers.md:354]:
```suggestion
保持只有一个 Issue 待处理有助于减少困惑，避免在同一问题上发生重复劳动。
```

**Comment by tengqm** [comment]
Thanks for the review, @SataQiu. All feedbacks resolved.

**Pull request merged**